src/vuestic-theme/vuestic-components/vuestick-date-picker/VuesticDatePicker.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <vue-flatpickr-component
+    v-model="valueProxy"
+    v-on="$listeners"
+    :config="fullConfig"
+  />
+</template>
+
+<script>
+  import VueFlatpickrComponent from 'vue-flatpickr-component'
+
+  export default {
+    components: {
+      VueFlatpickrComponent,
+    },
+    props: {
+      value: {
+        required: true,
+      },
+      config: {
+        default: () => {
+          return {}
+        },
+      },
+    },
+    computed: {
+      valueProxy: {
+        get () {
+          return this.value
+        },
+        set (value) {
+          this.$emit('input', value)
+        },
+      },
+      fullConfig () {
+        return Object.assign({}, this.defaultConfig, this.config)
+      },
+      defaultConfig () {
+        return {
+          allowInput: true, // false triggers `readonly` on input, which disables label animation.
+          nextArrow: '<span aria-hidden="true" class="ion ion-ios-arrow-forward"></span>',
+          prevArrow: '<span aria-hidden="true" class="ion ion-ios-arrow-back"></span>',
+        }
+      },
+    },
+  }
+</script>
+
+<style lang="scss">
+  @import "../../../sass/mixins";
+  @import "../../../sass/variables";
+
+  $datepickerActive: $vue-green;
+  $datepickerBackground: #333333;
+  $datepickerText: $white;
+
+  $datepickerToday: lighten($datepickerBackground, 5);
+  $datepickerOtherMonth: darken($datepickerText, 40);
+  $datepickerWeekday: darken($datepickerText, 50);
+  $datepickerDayHover: darken($datepickerText, 70);
+  $datepickerDayRange: $almost-black;
+  $datepickerSeparatorColor: $almost-black;
+
+  $borderPadding: 0.5rem;
+  $dayPadding: 0.375rem;
+  $daySize: 1.75rem;
+
+  .flatpickr-calendar {
+    background-color: $datepickerBackground;
+    border-radius: 0;
+
+    &.arrowTop {
+      &:before {
+        border-bottom-color: $datepickerBackground;
+      }
+      &:after {
+        border-bottom-color: $datepickerBackground;
+      }
+    }
+
+    &.arrowBottom {
+      &:before {
+        border-top-color: $datepickerBackground;
+      }
+      &:after {
+        border-top-color: $datepickerBackground;
+      }
+    }
+
+    .flatpickr-days {
+      width: $daySize * 7 + $dayPadding * 6 + $borderPadding * 2;
```

Comment:
![screen shot 2018-04-27 at 4 55 04 pm](https://user-images.githubusercontent.com/3674348/39366602-113ac4b0-4a3d-11e8-891c-fe802ecb9e5b.png)
padding right is strange